voluseg/_tools/load_parameters.py

```diff
@@ -15,12 +16,13 @@ def load_parameters(filename: str) -> dict:
     dict
         Parameters dictionary.
     """
-    try:
+    if filename.split(".")[-1] == "pickle":
         with open(filename, "rb") as file_handle:
             parameters = pickle.load(file_handle)
-
-        print("parameter file successfully loaded.")
-        return parameters
-
-    except Exception as msg:
-        print("parameter file not loaded: %s." % (msg))
+    elif filename.split(".")[-1] == "json":
+        with open(filename, "r") as file_handle:
+            parameters = json.load(file_handle)
```

Comment:
Can we double check that parameters returned by `json` and `pickle` are in exactly the same format? This could also be added as a unit test.

Reply:
Suggest to add this code for json

```
# Convert lists back to numpy arrays
for key, value in parameters.items():
    if isinstance(value, list):
        parameters[key] = np.array(value)
```